packages/components/atoms/f-form-field/src/components/FormField.vue

```diff
@@ -44,7 +44,8 @@
                 v-bind="$attrs"
                 :class="[
                     $style['c-formField-field'],
-                    $style['c-formField-field--focus']
+                    $style['c-formField-field--focus'],
+                    $style['c-formField-field--textarea']
```

Comment:
Not related to this PR (will still approve) but do we need the `c-formField-field--focus` class?

Seems like those styles should just be an extension of `c-formField-field` potentially (as it implies a state that doesn't exist)

Reply:
There is a condition on input element to add focus class only if `isSelectionControl=true` so focus styles were separated into their own class

Reply:
Just looking over the old PR (which is actually mine), I think it's an error that I put in a while ago.

I'll update this and get a PR in, but I think the naming is wrong as I may have missed the `!` on [line 234](https://github.com/justeat/fozzie-components/pull/964/files#diff-0c855ee5f708709123fe3e0cd37f1060278c6ead61d3a62a053e5c23b45a03b3R234) – which is why this is confusing me!